Description: Debian uses different text domain for the lang files
Forwarded: not-needed
Author: D Haley <mycae - gmx - com>
--- 3depict-0.0.16.orig/src/3Depict.cpp
+++ 3depict-0.0.16/src/3Depict.cpp
@@ -166,7 +166,7 @@
 		else
 		{
 			//Set the gettext language
-			textdomain( PROGRAM_NAME );
+			textdomain( "3depict");
 			setlocale (LC_ALL, "");
 #ifdef __WXMAC__
 			bindtextdomain( PROGRAM_NAME, paths->GetResourcesDir().mb_str(wxConvUTF8) );
@@ -198,8 +198,8 @@
 					break;
 			}			
 #else
-			bindtextdomain( PROGRAM_NAME, "/usr/share/locale" );
-			bind_textdomain_codeset(PROGRAM_NAME, "utf-8");
+			bindtextdomain( "3depict", "/usr/share/locale" );
+			bind_textdomain_codeset("3depict", "utf-8");
 #endif
 		}
 	}
